_collections/_news/2025-06-24-year-in-review-2025.md

```diff
@@ -0,0 +1,231 @@
+23rd of March, giving each team a full 24 hours to create something from scratch with the theme "Building Foundations" - 
+to which those teams most certainly delivered.
+
+![The birmingHack logo with the date included.](/assets/images/2025-end-of-year-newsletter/birminghack.png)
+
+There are numerous other challenges available throughout the event, such as extra categories teams could aim for other
+than the main theme, such as GDS' "Best UI/User Experience" category, AlgoSoc's "Best AI Hack without using an LLM" (which
+produced AI hacks that I was surprised to hear were made within the 24 hours!) and Majestic's "Surprise Me!" category.
+However, there were also bonus challenges that were separate to the main hacking - such as AFNOM's CTF, a cup-stacking
+challenge and AlgoSoc's hilarious meme competition. To give a break from hacking, we held a lot of smaller break events, 
+such as a midnight movie event, workshops for Git and the Neuphonic platform (one of our sponsors for the event), and 
+general food breaks for breakfast, lunch and dinner.
+
+![A photo of UG04 and attendees working on their projects.](/assets/images/2025-end-of-year-newsletter/birminghack-ug04.jpg)
+
+There were several memorable projects submitted with different purposes, inspirations and skill levels, that all reflected
+the individual teams that pulled everything together. Some teams focused on projects that would help others, such as
+Speaking With Claire, developed by a team whose first language wasn't English, but required to pass an English exam to
+enter university; No Room For Doom, an online healthcare service for citizens in Sudan, a country heavily torn by war
+and has a lack of healthcare services available; and of course, Too Bad To Keep, a response to the ongoing bin strikes
+in Birmingham where drivers can collect waste from those who cannot dump it at a landfill, and get paid for it. There were
+some more light-hearted games developed alongside those projects, such as Clunks Foundation Creation (very on the nose, 
+guys), Notable and Secret Sandwich Service, in addition to the Spinning Raps, of which won CSS' own category for
+most fun hack! And of course, there was a significant focus on AI within some projects, with Neuphonic providing their
+platform for free to all birmingHack participants and AlgoSoc hosting its AI hack category, which produced some incredibly
+interesting results - SocratEase and Ocular are two of these projects, where SocratEase would turn speech videos of
+yourself into feedback provided by AI, and Ocular used facial detection at your front door to ask those coming to your
+house who was visiting, their purpose of visiting, and whether it recognised that person from an existing contact. Two
+other projects that won prizes include StudySync, which connects students to study together, and Guess the Voice, which
+used Neuphonic's API to use celebrity voices, but also won in GDS' category for being the most user-friendly project.
```

Comment:
this para is very dense, consider fewer of them or just discussing the two you wanted to highlight in the next para (ThatsAwkward and FLOORPLAN) and linking the rest to devpost